sdcm/tester.py

```diff
@@ -3172,18 +3171,29 @@ def fixture_event_system(self, setup_logging):
         yield
         self.stop_event_device()
 
+    @pytest.fixture(autouse=True, name='threads_checks', scope='session')
+    def fixture_threads_checks(self):
```

Comment:
shouldn't be `self fixture_threads_checks(self, event_system):` to make sure it runs last?

Reply:
the scope make it run last, event_system is on function scope, you can't mix fixture of different scopes